CHANGELOG.md

```diff
@@ -7,8 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Support for multiline string literals within compiler directives.
+- Support for the `TEXTBLOCK` directive.
+- **API:** `CompilerDirectiveParser` can now return a new `TextBlockDirective` type.
+- **API:** `CheckVerifier::withCompilerVersion` method.
+- **API:** `CheckVerifier::withToolchain` method.
```

Comment:
Include new API method `DelphiFile::getTextBlockLineEndingModeRegistry`.

Reply:
I actually don't want `DelphiAst::getDelphiFile` to be exposed in the API because `DelphiFile` isn't in the custom rules API. Its API surface has a lot of overlap with `DelphiCheckContext`, and it feels quite inappropriate to go to the root of the AST to get some additional metadata about the source file.

Instead of doing this, I'll be deprecating `DelphiAst::getDelphiFile`.